sky/serve/core.py

```diff
@@ -336,6 +345,12 @@ def update(task: 'sky.Task', service_name: str) -> None:
         with ux_utils.print_exception_no_traceback():
             raise ValueError('Service section not found in the task. ')
 
+    if task.service.spot_placer is not None:
```

Comment:
should we print some warning to suggest update spot policy/spot placer is not supported now

Reply:
https://github.com/skypilot-org/skypilot/blob/serve-spot/sky/serve/autoscalers.py#L396-L400
It is supported. We probably need some tests to see if there might be unexpected behavior. Since old replicas will be scaled down, I would imagine this to be fine.

Reply:
Oh sry I mean autoscaler. IIUC this is not supported?

Reply:
If we only expose `spot_policy` to the user then it should be fine

Reply:
yeah updating autoscaler is not supported.

Reply:
I think we can still allow user to update autoscaler. but can print out log that it is not recommended.

Reply:
Why do you think we should not recommend updating autoscaler?

Reply:
Might be fine. I am just not sure.